qulice-pmd/src/test/resources/com/qulice/pmd/PlainJUnitAssertionImportBlock.java

```diff
@@ -0,0 +1,12 @@
+package com.qulice.pmd;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
@vkuchyn static imports are prohibited, so you didn't need to handle that, now you can leave it, but you wasted some of your time